core/web-assets/src/main/assets/js/apps/onms-daemons/index.js

```diff
@@ -0,0 +1,106 @@
+const elementList = require('../onms-elementList/lib/elementList');
+require('../../lib/onms-pagination');
+require('../../lib/onms-http');
+require('angular-bootstrap-confirm');
+require('angular-bootstrap-toggle/dist/angular-bootstrap-toggle');
+require('angular-bootstrap-toggle/dist/angular-bootstrap-toggle.css');
+require('angular-ui-router');
+
+const indexTemplate = require('./index.html');
+
+(function () {
+    'use strict';
+
+    var MODULE_NAME = 'onms.daemons';
+
+    angular.module(MODULE_NAME, [
+        'angular-loading-bar',
+        'ngResource',
+        'ui.router',
+        'ui.bootstrap',
+        'ui.checkbox',
+        'ui.toggle',
+        'onms.http',
+        'onms.elementList',
+        'mwl.confirm',
+        'onms.pagination'
+    ])
+        .config(['$locationProvider', function ($locationProvider) {
+            $locationProvider.hashPrefix('!');
+            $locationProvider.html5Mode(false);
+        }])
+        .config(['$stateProvider', function ($stateProvider) {
+            $stateProvider
+                .state('daemons', {
+                    url: '',
+                    controller: 'ListController',
+                    templateUrl: indexTemplate
+                });
+        }])
+        .controller('ListController', ['$scope', '$http', '$timeout', function ($scope, $http, $timeout) {
+            $scope.serverUrl = location.protocol + '//' + location.hostname+(location.port ? ':'+location.port: '');
+
+
+            $scope.daemons = [];
+
+
+            $scope.LookupDaemonState = function (daemon) {
+                if (daemon.reloadCount > 5) {
+                    daemon.isReloading = false;
+                    daemon.reloadNote = "Undefined";
+                    daemon.reloadIconClass = "glyphicon glyphicon-question-sign";
+                    return;
+                }
+                daemon.reloadCount = daemon.reloadCount + 1;
+
+                $http.get('rest/daemons/checkReloadState/' + daemon.name + '/').then(function (response) {
+                    if (response.data.reloadState === "Success") {
+                        daemon.isReloading = false;
+                        daemon.reloadNote = response.data.reloadState;
+                        daemon.reloadIconClass = "glyphicon glyphicon glyphicon-ok-sign";
+                    } else if (response.data.reloadState === "Failed") {
+                        daemon.isReloading = false;
+                        daemon.reloadNote = response.data.reloadState;
+                        daemon.reloadIconClass = "glyphicon glyphicon glyphicon-remove-sign";
+                    } else {
+                        $timeout($scope.LookupDaemonState, 1000, true, daemon);
+                    }
+                });
+            };
+
+            $scope.doTheCopy = function (daemon) {
+                //MAGIC
+            };
+
+            $scope.reloadPressed = function (daemon) {
+                $http.post('rest/daemons/reload/' + daemon.name + '/')
```

Comment:
Same as above, this may be better at `rest/daemons/<name>/reload` in order to be more rest like.